### ISSUE TITLE
[Bug]: Issue with Confirmatory Factor Analysis

### ISSUE THREAD

**Issue by LennyBee** [jasp-stats/jasp-issues#1541]
### JASP Version

0.16

### JASP Module

jaspSem

### What analysis are you seeing the problem on?

Confirmatory factor analysis

### What OS are you seeing the problem on?

Windows 10 Home 20H2

### Bug Description

I ask JASP to plot the CFA Model and I get this error

```
This analysis terminated unexpectedly.

Error in bars[[node]] <- pnorm(GroupThresh$est[GroupThresh$lhs = GroupVars$name[node]]): attempt to select more than one element in integerOneIndex

Stack trace
tryCatchOne(expr, names, parentenv, handlers[[1]])

doTryCatch(return(expr), name, parentenv, handler)

withCallingHandlers(expr = analysis(jaspResults = jaspResults, dataset = dataset, options = options), error = .addStackTrace)

analysis(jaspResults = jaspResults, dataset = dataset, options = options)

ConfirmatoryFactorAnalysis(...)

.cfaPlotPath(jaspResults, options, cfaResult)

jaspBase:::.suppressGrDevice(pathplot <- semPlot::semPaths(object = .cfaLavToPlotObj(cfaResult[['lav']], options), DoNotPlot = TRUE, ask = FALSE, layout = 'tree', intercepts = options$plotmeans, whatLabels = if (!options$plotpars) 'name' else if (options$plotstd) 'std' else 'par', mar = ifelse(rep(is.null(options$secondOrder), 4), c(12, 3, 12, 3), c(6, 3, 6, 3)), edge.color = 'black', color = list(lat = '#EAEAEA', man = '#EAEAEA', int = '#FFFFFF'), border.width = 1.5, edge.label.cex = 0.9, lty = 2,
    title = FALSE))

eval(plotFunc, parent.frame())

eval(plotFunc, parent.frame())

semPlot::semPaths(object = .cfaLavToPlotObj(cfaResult[['lav']], options), DoNotPlot = TRUE, ask = FALSE, layout = 'tree', intercepts = options$plotmeans, whatLabels = if (!options$plotpars) 'name' else if (options$plotstd) 'std' else 'par', mar = ifelse(rep(is.null(options$secondOrder), 4), c(12, 3, 12, 3), c(6, 3, 6, 3)), edge.color = 'black', color = list(lat = '#EAEAEA', man = '#EAEAEA', int = '#FFFFFF'), border.width = 1.5, edge.label.cex = 0.9, lty = 2, title = FALSE)

To receive assistance with this problem, please report the message above at: https://jasp-stats.org/bug-reports
```

### Expected Behaviour

I should have seen the CFA model

### Steps to Reproduce

1. Unclick the button model plot
2. Reclick the button model plot
3. 
...

### Final Checklist

- [X] I have included a screenshot showcasing the issue, if any.
- [X] I have included a JASP file that causes the crash/bug, if any.
- [X] I have accurately described the bug, and steps to reproduce it.

**Comment by boutinb** [comment]
Hi @LennyBee Could you send a JASP file showing this error? For this zip the file and drag & drop it in a new comment of this GitHub issue.

**Comment by Kucharssim** [comment]
Dear @LennyBee,

thank you for submitting this issue! We were already made aware of it previously (#1548) and fixed it since then (jasp-stats/jaspFactor#69). The fixed version of the analysis will be available in the next JASP release (which should be out soon).

I am closing this issue. Please feel free to reopen the issue in case the problem persists in the future versions.
